K2Bridge/Visitors/Aggregations/HistogramAggregationVisitor.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Visitors
+{
+    using System;
+    using System.Text;
+    using K2Bridge.Models.Request.Aggregations;
+    using K2Bridge.Models.Response;
+    using K2Bridge.Utils;
+
+    /// <content>
+    /// A visitor for the <see cref="HistogramAggregation"/> element.
+    /// Sample KustoQL Query:
+    /// | where ['AvgTicketPrice'] >= bin(50,20) and ['AvgTicketPrice'] < bin(150,20)+20
```

Comment:
This query does no longer match the full query below

Reply:
Done